mariadb/run.sh

```diff
@@ -26,42 +21,42 @@ while ! mysql -e "" 2> /dev/null; do
     sleep 1
 done
 
-echo "[INFO] Check data integrity and fix corruptions"
+bashio::log.info "Check data integrity and fix corruptions"
 mysqlcheck --no-defaults --check-upgrade --auto-repair --databases mysql --skip-write-binlog > /dev/null || true
 mysqlcheck --no-defaults --all-databases --fix-db-names --fix-table-names --skip-write-binlog > /dev/null || true
 mysqlcheck --no-defaults --check-upgrade --all-databases --auto-repair --skip-write-binlog > /dev/null || true
 
 # Init databases
-echo "[INFO] Init custom database"
-for line in $DATABASES; do
-    echo "[INFO] Create database $line"
+bashio::log.info "Init custom database"
+for line in bashio::config "databases"; do
```

Comment:
This will not work this way. You need to execute this in a subshell.
Please make sure you test the commits you make ;)